From 1a80dd8b7bb0b3228a773f47be5006583950f1d6 Mon Sep 17 00:00:00 2001
From: Qin Liu <lqgy2001@gmail.com>
Date: Thu, 1 Jun 2023 10:17:27 +0800
Subject: [PATCH 36/49] update search-index

1. improve search performance for memory mode
2. reduce memory usage by reserve vector
---
 cmake/autogenerated_myscale_versions.txt | 6 +++---
 contrib/search-index                     | 2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/cmake/autogenerated_myscale_versions.txt b/cmake/autogenerated_myscale_versions.txt
index 4333f67496..c3d6437819 100644
--- a/cmake/autogenerated_myscale_versions.txt
+++ b/cmake/autogenerated_myscale_versions.txt
@@ -4,8 +4,8 @@
 # only DBMS_TCP_PROTOCOL_VERSION should be incremented on protocol changes.
 SET(MYSCALE_VERSION_MAJOR 0)
 SET(MYSCALE_VERSION_MINOR 9)
-SET(MYSCALE_VERSION_PATCH 2)
-SET(MYSCALE_VERSION_DESCRIBE myscale-v0.9.2)
-SET(MYSCALE_VERSION_STRING 0.9.2)
+SET(MYSCALE_VERSION_PATCH 3)
+SET(MYSCALE_VERSION_DESCRIBE myscale-v0.9.3-testing)
+SET(MYSCALE_VERSION_STRING 0.9.3)
 # end of autochange
 
diff --git a/contrib/search-index b/contrib/search-index
index bea7a83da2..05b1f7b749 160000
--- a/contrib/search-index
+++ b/contrib/search-index
@@ -1 +1 @@
-Subproject commit bea7a83da2e16cd5f42dfd28c140eab08d7e77aa
+Subproject commit 05b1f7b749217a66b772852c9e82501a2f324488
-- 
2.32.1 (Apple Git-133)

